composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/form": "^2.3.5 || ^3.0",
         "symfony/http-foundation": "^2.3 || ^3.0",
         "symfony/property-access": "^2.3 || ^3.0",
+        "symfony/security": "^2.3 || ^3.0",
```

Comment:
It's a test that can be used outside the bundle (as explained in my commit message :P). BTW, this is why I said it's necessary to merge this if we want https://github.com/sonata-project/SonataAdminBundle/pull/4216 to be green one day